### ISSUE TITLE
Support svelte and sveltekit framework

### ISSUE THREAD

**Issue by smcenlly** [wallabyjs/console-ninja#19]
From product hunt: 

> can you support svelte and sveltekit framework? the JS code is inside the script tag (I don't know why isn't working). thanks a lot if you solve it wallaby!

**Comment by philsherry** [comment]
@ArtemGovorov Rather than opening a new issue, I'll just tag onto this one in case it helps with the Svelte case.

I’d previously noticed that the extension added this in my `.bashrc` file:
```sh
PATH=~/.console-ninja/.bin:$PATH
```

I’d taken it out, tidied it up and moved it into my `.path` file, which is called before `.bashrc` — I like to keep things tidy, and know what’s where and why. 
```sh
PATH="${HOME}/.console-ninja/.bin:${PATH}"
```

This `.path` file gets included in both bash _and_ zsh shells on my Macs. However, your addition is only appearing in `.bashrc` — despite my use of zsh at the time, which is my default shell. 

I noticed it had returned at some point, so I did some digging and it writes it back when I open a Svelte project. I cleaned out my `trace.log` file and opened a current Svelte project so that I could paste the contents here for you, but I had to remove the `filesToInstrument` array for security reasons. 

How are you targeting where and how to add this, and how to check whether it currently exists in the `$PATH`?

Hope this helps in some way.

```json
{
  "extensionFolder": "/Users/philsherry/.vscode/extensions/wallabyjs.console-ninja-0.0.113",
  "runtimeHookFile": "/Users/philsherry/.vscode/extensions/wallabyjs.console-ninja-0.0.113/out/runtimeHook/tracer.js",
  "globalErrorHandlerFile": "/Users/philsherry/.vscode/extensions/wallabyjs.console-ninja-0.0.113/out/runtimeHook/errorHandler.js",
  "filesToInstrument": [],
  "debug": false,
  "config": {
    "dateTimeFormat": "hh:mm:ss.SSS",
    "openLinksInEditor": true,
    "maxLogViewerEntries": 15000,
    "fontSize": 10,
    "outputMode": "Beside File",
    "captureFunctions": false,
    "enableProFeatures": true,
    "toolsToAutoPatch": [
      "vite",
      "jest",
      "webpack",
      "next.js",
      "cypress",
      "http-server",
      "serve",
      "nuxt",
      "remix",
      "qwik",
      "hydrogen"
    ],
    "showErrors": true,
    "showLogsOnlyFromOpenedFiles": false,
    "showLogsFromLogPointsOnly": false
  },
  "allowedHosts": [
    "localhost",
    "127.0.0.1",
    "example.cypress.io",
    "imac-5k",
    "192.168.1.64",
    "192.168.1.116"
  ],
  "host": "127.0.0.1"
}
```

**Comment by NikGovorov** [comment]
We simply check if it's in `.bashrc` or not. You may comment the line and leave it there then console-ninja will stop changing the file.

```
# PATH=~/.console-ninja/.bin:$PATH
```

**Comment by NikGovorov** [comment]
Taking into account that you don't even need the feature(https://marketplace.visualstudio.com/items?itemName=WallabyJs.console-ninja#universal-node-applications), I think it's reasonable to have a setting that will prevent automatic global PATH integration.

**Comment by NikGovorov** [comment]
We've implemented the setting in v0.0.115.

![image](https://user-images.githubusercontent.com/129254/235574169-e126be2c-cf63-4b20-9d61-23b4ee1d3018.png)

**Comment by texastoland** [comment]
Can you document this please 🙏🏼 It was tricky to find!
